charts/argo/templates/workflow-controller-config-map.yaml

```diff
@@ -51,4 +56,4 @@ data:
 {{ toYaml .Values.controller.persistence | indent 6 }}{{- end }}
     {{- if .Values.controller.workflowDefaults }}
     workflowDefaults:
-{{ toYaml .Values.controller.workflowDefaults | indent 6 }}{{- end }}
+{{ toYaml .Values.controller.workflowDefaults | indent 6 }}{{- end }}
```

Comment:
The newline should stay, I think the problem is that above you are missing an {{- end }}

Reply:
Oops. Fixed. Added {{-end}} and reverted commit to remove end line